doc/source/extending-jupyterhub.rst

```diff
@@ -85,6 +85,20 @@ For more information see the full example of Google OAuth2 in the next section.
           clientSecret: "an0ther1ongs3cretstr1ng"
           callbackUrl: "http://<your_jupyterhub_host>/hub/oauth_callback"
 
+**Globus**
+
```

Comment:
maybe add something short/descriptive so folks who don't know about Globus get a bit of context, e.g. "Globus support for efficient data transfer, for more details see <link to globus info>".

Reply:
Great idea! I'll add it.